apps/postgres-new/app/api/databases/[id]/upload/route.ts

```diff
@@ -1,30 +1,83 @@
 import { S3Client } from '@aws-sdk/client-s3'
 import { Upload } from '@aws-sdk/lib-storage'
-import { NextRequest } from 'next/server'
+import { NextRequest, NextResponse } from 'next/server'
 import { createGzip } from 'zlib'
 import { Readable } from 'stream'
+import { createClient } from '~/utils/supabase/server'
+import { createScramSha256Data } from 'pg-gateway'
+import { generateDatabasePassword } from '~/utils/generate-database-password'
 
-const wildcardDomain = process.env.WILDCARD_DOMAIN ?? 'db.example.com'
+const wildcardDomain = process.env.NEXT_PUBLIC_WILDCARD_DOMAIN ?? 'db.example.com'
```

Comment:
Thinking now that it's probably better to let these throw an error than fallback to an arbitrary value

Reply:
Yep, we can use Zod for the env variables validation and loading as well. I'll add that to the issue about validation.